classes/fields/date.php

```diff
@@ -166,6 +166,19 @@ public function schema( $options = null ) {
 		return $schema;
 	}
 
+	/**
+	 * {@inheritdoc}
+	 */
+	public function format_display( $options, $js = false ) {
+
+		if ( $js && 'custom' === pods_v( static::$type . '_type', $options, 'format' ) ) {
+			$format = $this->format_datetime( $options, $js );
+			return $this->convert_format( $format, array( 'source' => 'jquery_ui', 'type' => 'date' ) );
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 69b66c98be2ec647eca42e728d0140c4</sub>